webui/react/src/shared/configs/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es2020",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": false,
-    "skipLibCheck": false,
+    "skipLibCheck": true,
```

Comment:
This option was actually overridden in create react app to be false, so this is more restoring functionality that already existed. By enabling the library check, typescript throws an error based on the types in react.

Reply:
ah does the error come from table component? if so, that makes sense. we need to fix it at some point

Reply:
Can we just fix that type error?

Reply:
@keita-determined no, the errors are in the type definitions. You can see this for yourself by changing `skipLibCheck` to `false` locally and running `npx tsc`.

Reply:
Is that because the version of the type definitions and react library are mismatched?

Reply:
@ClaireNeveu No -- we've configured typescript not to check the type of plain javascript modules. These errors are in the type definitions themselves.

Reply:
@ClaireNeveu after poking at this for a few hours, I'm going to open a ticket for this task and push to handle this as a fast follow because getting the library types to work all the way through will cause a large expansion in scope. I was able to get most of the errors to clear by updating libraries, but there's a remaining issue in `@types/node` stemming from a difference in URL objects between browsers and nodejs that the swagger api library is bumping into. We might need to either do a complex find/replace as part of the binding generation or implement a custom codegen class, which is doable, but I'd want that change to be looked at separate from the other build issues.